CHANGELOG.md

```diff
@@ -7,6 +7,7 @@
 #### Fixes
 
 * [#546](https://github.com/ruby-grape/grape-swagger/pull/546): Move development dependencies to Gemfile - [@olleolleolle](https://github.com/olleolleolle).
+* [#547](https://github.com/ruby-grape/grape-swagger/pull/547): Use entity_name event if type come from a string - [@frodrigo](https://github.com/frodrigo).
 
```

Comment:
ok @dblock … think now it would be nice to know how it should looks like …
sorry, but yhe syntax of the changelog hasn't my first priority 😉